src/main/java/teammates/logic/core/CoursesLogic.java

```diff
@@ -821,6 +970,12 @@ public boolean isCourseArchived(CourseAttributes course, InstructorAttributes in
         return result;
     }
     
+    /**
+     * Returns a list of courseIds for all archived courses for all instructors.
```

Comment:
@JosephineKwa @damithc Moved all the one line descriptions to return tag where ever possible to avoid duplication.